src/gateway/game.gateway.ts

```
@@ -1,5 +1,4 @@
-// game.gateway.ts
-import { forwardRef, Inject, UseGuards } from '@nestjs/common';
+import { forwardRef, HttpException, Inject, UseGuards } from '@nestjs/common';
```

ESLint

[error] 'HttpException' is defined but never used.

(@typescript-eslint/no-unused-vars)